app/models/log.rb

```diff
@@ -203,8 +203,4 @@ def reset_location(is_inferred, la_key, is_inferred_key, postcode_key, is_la_kno
     self[is_inferred_key] = false
     self[postcode_key] = nil
   end
-
-  def format_as_currency(num_string)
-    ActionController::Base.helpers.number_to_currency(num_string, unit: "£")
-  end
```

Comment:
Oh I see you're removing this. I believe it's being called a couple of times from in the sales log?